gtsfm/evaluation/compare_metrics.py

```diff
@@ -0,0 +1,60 @@
+"""Script to log metrics for comparison across SfM pipelines (GTSfM, COLMAP, etc.) as a JSON.
+
+Authors: Jon Womack
+"""
+import os
+import json
+from typing import Dict
+
+from gtsfm.evaluation.metrics import GtsfmMetric, GtsfmMetricsGroup
+
+
+
+def compare_metrics(txt_metric_paths: Dict[str, str], json_path: str) -> None:
+    """
+
+    Args:
+        txt_metric_paths: a list of paths to directories containing: cameras.txt, images.txt, and points3D.txt files
+    """
+    gtsfm_metrics = []
+    for pipeline_name in txt_metric_paths.keys():
+        # Add 3D point information
+        fpath = os.path.join(txt_metric_paths[pipeline_name], "points3D.txt")
+        with open(fpath, "r") as f:
+            data = f.readlines()
+            points = data[3:]
+            track_lengths = []
+            for point in points:
+                track_lengths.append(len(point.split()[8:]) / 2)
+
+        gtsfm_metrics.append(GtsfmMetric('Number of 3D Points ' + pipeline_name,
```

Comment:
can we reformat this file with `python black`?

Reply:
Done